DSL/Ruuter/GET/services/services-detailed/nok.yml

```diff
@@ -12,27 +12,27 @@ getFaults:
   args:
     url: "[#SERVICE_OPENSEARCH]/ruuterlog/_search"
     query:
-      size: 20
+      from: ${(incoming.params.page - 1) * incoming.params.page_size}
+      size: ${incoming.params.page_size}
       _source_excludes: "stackTrace,statusCode"
     body:
+      sort: [{ "timestamp": { "order": "${incoming.params.order}" } }]
```

Comment:
Please add the new parameters to the  `declaration` step under `allowlist` > `params`, like so:
```
declaration:
  ...
  allowlist:
    params:
      - field: page_size
        type: number
        description: "Parameter 'page_size'"
      - field: sorting
        type: string
        description: "Parameter 'sorting'"
      ...etc
```